users/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import json
+import re
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+from my_settings import SECRET_KEY
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email          = data['email']
+            password       = data['password']
+            address        = data['address']
+            name           = data['name']
+            ck_password    = data['ck_password']
+            phone_number   = data['phone_number']
+            Valid_password = re.compile('^(?=.*[a-zA-Z])(?=.*[\d])(?=.*[~!@#$%^&*_+])[a-zA-Z\d~!@#$%^&*_+]{8,}$')
+            Valid_email    = re.compile('^[a-zA-Z\d+-_.]+@[a-zA-Z\d]+\.[a-zA-Z\d+-.]+$')
```

Comment:
정규식은 아래와 같이 `regex`라는 용어를 많이 붙여서 사용합니다.

```suggestion
            REGEX_EMAIL = re.compile('^[a-zA-Z\d+-_.]+@[a-zA-Z\d]+\.[a-zA-Z\d+-.]+$')
            REGEX_PASSWORD = re.compile('^(?=.*[a-zA-Z])(?=.*[\d])(?=.*[~!@#$%^&*_+])[a-zA-Z\d~!@#$%^&*_+]{8,}$')
```

Reply:
regex로 수정완료했습니다.